analyzers/PapasEventFromRoot.py

```diff
@@ -0,0 +1,62 @@
+from heppy.framework.analyzer import Analyzer
+from heppy.papas.data.papasevent import PapasEvent
+from heppy.papas.graphtools.DAG import Node
+
+class PapasEventFromRoot(Analyzer):
```

Comment:
all reading stuff should be put in the reader, or we force people to add this second analyzer in their sequence. 
As for other collections in the reader: 
if rec is specified: get rec
if gen_rec_links is specified: get links, and do the linking. 
you may add a private method to the reader if you so with, e.g. _set_rec_to_gen_ptc_links